### DISCUSSION TITLE
Add FM WIDE option in Demodu menu (FMW)

### DISCUSSION THREAD

**Discussion by brookbonde** [egzumer/uv-k5-firmware-custom#190] [Ideas]
Please add an FM WIDE (FMW) option, alongside the FM, AM, USB in the Demodu menu.  This would allow direct reception of the FM broadcast band from within the main VFO and would also utilise the S-meter in the FM VHF broadcast band.  It would also allow for reception of other low power devices that broadcast in FM wide, such as wireless headphones on 863 MHz FM wide (for example).

**Comment by chouyu** [comment]
FM broadcast and radio use two different chips.

**Comment by ErikS-web** [comment]
See blockdiagram (right corner)
https://github.com/ludwich66/Quansheng_UV-K5_Wiki/wiki/Block_Diagram

This firmware modifies original range 50 - 600 MHz to now 18 MHz ~ 660 MHz and 840 ~ 1300 MHz.
So **NO 863 Mhz.**

The use of this transceiver is printed on the sticker of the package.
We try to extend, but there are limits.

Suggest you look for other type of radio

**Reply by brookbonde** [reply]
Of course it receives at 863 MHz.  This is between 840 ~ 1300 MHz as you quoted above.
I have even had the signal generator on it to check RX.  I can also receive local wireless headphones in the 863 MHz band, but I am unable to select FM Wide in the band.

**Comment by ErikS-web** [comment]
The FM radio is an[ BK1080](https://file2.dzsc.com/icpdf/17/12/18/1125865_165859249.pdf), ,that support 64~108 MHz band. NO more.

Where did you find info that the FM WIDE (FMW) is possible with the BK4819 ?

[BK4819.PDF](https://github.com/egzumer/uv-k5-firmware-custom/files/13590097/BK4819.PDF)

**Reply by brookbonde** [reply]
If you are confirming that FM wide cannot be enabled outside of the BK1080 frequency coverage, then that’s fine. Clearly a hardware limitation. This thread can now to closed. Thank you.
